R/sxp.R

```diff
@@ -52,13 +52,18 @@
 #' sxp(e1, expand = "environment")
 #' sxp(e2, expand = "environment")
 sxp <- function(x, expand = character(), max_depth = 5L) {
-
   opts <- c("character", "altrep", "environment", "call", "bytecode")
   if (any(!expand %in% opts)) {
-    abort("`expand` must contain only values from ", paste("'", opts, "'", collapse = ","))
+    abort((
+      paste(
+        "`expand` must contain only values from:",
+        paste0("'", opts, "'", collapse = ", ")
+      )
+    ))
   }
 
-  obj_inspect_(x,
+  obj_inspect_(
```

Comment:
Would you mind not restyling in this PR? It's better to fix code style in a separate PR so it's easy to focus on the substantive changes.

Reply:
Yes I thought the same thing after I saw it on github - I'll fix it and add a test.

Reply:
@hadley I have added the test and removed the restyling (format on save was the culprit!). Hope this does the job now - if not let me know.